web-components/src/components/button/scss/settings.scss

```diff
@@ -222,7 +222,7 @@ $button--duck-egg__border: 0 0 0 1px $md-gray-50 !default;
 $button--violet__background-color: $md-violet-50 !default;
 $button--violet__background-color--hover: $md-violet-60 !default;
 $button--violet__background-color--active: $md-violet-70 !default;
-$button--violet__border: 0 0 0 1px $md-violet-90 !default;
```

Comment:
@helen-v Is this card related or responding to another issue?

Reply:
Only here noticed that the outline border for violet does not match .